index.d.ts

```diff
@@ -14,6 +14,10 @@ export interface LoopReducer<S, A extends Action> {
   (state: S | undefined, action: AnyAction, ...args: any[]): S | Loop<S, A>;
 }
 
+export interface NonInitialLoopReducer<S, A extends Action> {
```

Comment:
I'm not quite happy with the naming of this interface. Suggestions for better names are welcome.

Reply:
LoopReducerWithDefinedState?

Reply:
Yeah, that does feel more consistent. I'll go ahead and change it.